lookout/templates/dummy-analyzer-deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "lookout.fullname" . }}-dummy-analyzer
+  labels:
+    app: dummy-analyzer
+    chart: {{ template "lookout.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount.dummyAnalyzer }}
+  selector:
+    matchLabels:
+      app: dummy-analyzer
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: dummy-analyzer
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: dummy-analyzer
+          image: "{{ .Values.image.dummyAnalyzer.repository }}:{{ .Values.image.dummyAnalyzer.tag }}"
+          imagePullPolicy: {{ .Values.image.dummyAnalyzer.pullPolicy }}
+          args:
+            - serve
+            - {{ .Values.app.lookout.repository }}
```

Comment:
analyzer doesn't need repository argument. It can work with any repo sent by lookout server.

Reply:
fixed in a3951ba